adapters/Maio/MaioAdapter/GADMMaioConstants.h

```diff
@@ -17,10 +17,16 @@ static NSString *const kGADMMaioSDKErrorDomain = @"com.google.mediation.maioSDK"
 /// maio mediation network adapter version.
 static NSString *const kGADMMaioAdapterVersion = @"1.5.6.1";
 
+/// maio publisher ID.
+static NSString *const kGADMMaioAdapterPublisherID = @"publisherId";
+
 /// maio media ID.
 static NSString *const kGADMMaioAdapterMediaId = @"mediaId";
 
 /// maio zone ID.
 static NSString *const kGADMMaioAdapterZoneId = @"zoneId";
 
+/// Maio zoneID for OpenBidding
+static NSString *const kGADRTBMaioAdapterZoneId = @"dummyZoneForRTB";
```

Comment:
Does this mean that the Zone ID is not required for open bidding?
If so, maybe we can use this as a literal when requesting for open bidding and remove this constant.
Additionally, add a comment saying zone ID is not required for open bidding.

Reply:
> Does this mean that the Zone ID is not required for open bidding?

Yes. I understand the policy.